src/aws/osml/tile_server/viewpoint/viewpoint_id/map/__init__.py

```diff
@@ -0,0 +1,4 @@
+#  Copyright 2024 Amazon.com, Inc. or its affiliates.
+
+from .router import get_map_tilesets
+from .tileset import _invert_tile_row_index, get_map_tile, get_map_tileset_metadata
```

Comment:
Does _invert_tile_row_index need to be exported or is it a private function just used by get_map_tile?

Reply:
Cleaned up all the __init__.py files.